inference-engine/src/mkldnn_plugin/nodes/mkldnn_scatter_update_node.cpp

```diff
@@ -182,7 +180,7 @@ void MKLDNNScatterUpdateNode::initSupportedPrimitiveDescriptors() {
             !getParentEdgeAt(DATA_ID)->getParent()->isConstant();
```

Comment:
change previous line
add check on `!isDynamicNode()`

`bool canBeInplace = !isDynamicNode() && getParentEdgeAt(DATA_ID)->getParent()->getChildEdges().size() == 1 &&
            !getParentEdgeAt(DATA_ID)->getParent()->isConstant();`

Reply:
Corrected, thank you